openstack/cinder/files/api-paste.ini

```diff
@@ -58,5 +58,8 @@ admin_user = {{ salt['pillar.get']('cinder:admin_user', 'cinder') }}
 admin_password = {{ salt['pillar.get']('virl:password', salt['grains.get']('password', 'password')) }}
 auth_host = {{ salt['pillar.get']('virl:hostname', salt['grains.get']('hostname', 'virl')) }}
 auth_port = {{ salt['pillar.get']('cinder:auth_port', '35357') }}
-auth_uri = http://{{ salt['pillar.get']('virl:hostname', salt['grains.get']('hostname', 'virl')) }}:5000/v2.0
+auth_uri = http://{{ salt['pillar.get']('virl:hostname', salt['grains.get']('hostname', 'virl')) }}:5000/v3
```

Comment:
I don't like this policy of "do not fix unused files and let unsuspecting people be surprised later".  BTW it should have been parametrized with keystone_auth_version, all of this should have, this pull request was incomplete...